aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -272,6 +286,57 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    // Offer rotation capability of this account to another address
+    public entry fun offer_rotation_capability_ed25519(account: &signer, rotation_capability_sig_bytes: vector<u8>, pk_bytes: vector<u8>, recipient_address: address) acquires Account {
```

Comment:
Rename `pk_bytes` to `account_pk_bytes` to better indicate that this is the account's PK.